src/viur/core/modules/user.py

```diff
@@ -904,17 +905,26 @@ def add(self, otp=None):
 
         if otp is None:
             return self._user_module.render.second_factor_add(
-                tpl=self.otp_add_template,
+                tpl=self.second_factor_add_template,
```

Comment:
Btw: here we have to pass the `tpl` parameter because we are not inside the `user` module. But we are using the `user.Render`, where the `parent` is the `user` module. So `_choose_template` won't use the attribute of that class (where it logically belongs).